index.html

```diff
@@ -0,0 +1,161 @@
+            </p>
+            <p class="date-details">Mon, 01<sup>st</sup> August 2022</p>
+            <small class="venue">Online Trainings</small>
+            <div id="bg"></div>
+        </section>
+        <section id="about" class="about-page container-fluid">
+            <h2 class="about-header">Main Program</h2>
+            <div class="divider" id="div-1"></div>
+            <div class="activities container-fluid d-flex flex-column align-items-center justify-content-center">
+                <ul class="activity-list col-12">
+                    <li class="activity">
+                      <img src="./icons/block01.jpg" alt="">
+                      <span class="activity-name">Certified Blockchain Developer™</span>
+                      <span class="activity-description">
+                        Certified Blockchain & Supply Chain Professional™ training is specially 
+                        curated to guide an individual in creating solutions that can impact all the facets of the Supply Chain
+                      </span>
+                    </li>
+                    <li class="activity">
+                      <img src="./icons/block02.jpg" alt="">
+                      <span class="activity-name">Certified Metaverse Expert™</span>
+                      <span class="activity-description">
+                        The certification aims to provide a deeper understanding of the Metaverse and offer you a glimpse
+                         of the emerging future of the Web and the way we will interact with it.
+                      </span>
+                    </li>
+                    <li class="activity">
+                        <img src="./icons/block05.jpg" alt="">
+                        <span class="activity-name">Certified Cardano Expert™ </span>
+                        <span class="activity-description">
+                            Cardano, which is a fully open source decentralized public Blockchain and cryptocurrency, 
+                            is making impressive gains after the likes of Bitcoin and Ethereum.</span>
+                    </li>
+                    <li class="activity">
+                        <img src="./icons/block04.jpg" alt="">
+                        <span class="activity-name">Certified Ethereum Developer™</span>
+                        <span class="activity-description">
+                            An Ethereum developer is one who has expertise in one of the best decentralized 
+                            blockchain platforms available today, Ethereum. 
+                        </span>
+                    </li>
+                </ul>
+                <button type="button" id="sub-btn" class="subscribe-btn">
+                    Join blockchain-council
+                </button>
+                <small id="dt-btn"><a href="#">SEE THE WHOLE PROGRAM</a></small>
+            </div>
+        </section>
+    </main>
+    <footer>

```

Comment:
- [ ] Kindly align the logo image on the home page footer to match the original design. Try to make it a bit smaller and aligned to the left side.
- [ ] Also, apply the same request on the footer on the about page.
![Screenshot (4)](https://user-images.githubusercontent.com/47625407/182980387-bee368f1-340d-4c11-b6dc-01651b0058f0.png)